sdk/keyvault/keyvault-certificates/src/identifier.ts

```diff
@@ -4,26 +4,16 @@
 import { parseKeyvaultIdentifier } from "./generated/utils";
 
 /**
- * Valid collection names for Key Vault Certificate identifiers.
+ * Represents a Key Vault Certificate Id and its parsed contents.
  */
-export type KeyVaultCertificatesIdentifierCollectionName = "certificates" | "deletedcertificates";
-
-/**
- * Represents a Key Vault identifier and its parsed contents.
- */
-export interface ParsedKeyVaultCertificatesIdentifier {
-  /**
-   * The type of resource under Key Vault that this identifier is referring to.
-   */
-  collection: KeyVaultCertificatesIdentifierCollectionName;
-
+export interface KeyVaultCertificateId {
   /**
-   * The originally received identifier.
+   * The source ID that was parsed.
    */
-  id: string;
+  sourceId: string;
 
   /**
-   * The Key Vault Certificate unique identifier (a URL).
+   * The Key Vault Certificate unique Id (a URL).
```

Comment:
I thought the `sourceId` was the unique id and not the vaultUrl

Reply:
In Key Vault, the unique ID contains the Vault URL too! But they're not the same.

Reply:
Ah I see your point, I'll fix it.

Reply:
I pushed a fix for this! Let me know what you think.